x-pack/plugin/security/qa/service-account/src/javaRestTest/java/org/elasticsearch/xpack/security/authc/service/ServiceAccountIT.java

```diff
@@ -197,6 +200,36 @@ public class ServiceAccountIT extends ESRestTestCase {
             }
         }""";
 
+    @ClassRule
+    public static ElasticsearchCluster cluster = ElasticsearchCluster.local()
+        .nodes(2)
+        .module("analysis-common")
```

Comment:
Mainly for my own learning: I assume this means the test cluster now uses the `integ_test` distribution instead of `default`?

Reply:
Correct. The integ test distribution is used by default.